examples/DotNetCore/ConsoleApplication/Program.cs

```diff
@@ -1,93 +1,66 @@
-                                  .Register("Language", refreshAll: true)
-                                  .SetCacheExpiration(TimeSpan.FromSeconds(10));
-                       });
+﻿using Azure.Identity;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
 
-                // Get an instance of the refresher that can be used to refresh data
-                _refresher = options.GetRefresher();
-            });
+// Setup configuration builder
+var builder = new ConfigurationBuilder();
 
-            Configuration = builder.Build();
-        }
+// Load base configuration from appsettings.json and environment variables
+builder.AddJsonFile("appsettings.json")
+       .AddEnvironmentVariables();
 
-        private static async Task Run(CancellationToken token)
-        {
-            string display = string.Empty;
-            StringBuilder sb = new StringBuilder();
+// Build initial configuration
+var initialConfig = builder.Build();
 
-            do
-            {
-                sb.Clear();
-
-                // Trigger and wait for an async refresh for registered configuration settings
-                await _refresher.TryRefreshAsync();
-
-                sb.AppendLine($"{Configuration["AppName"]} has been configured to run in {Configuration["Language"]}");
-                sb.AppendLine();
-
-                sb.AppendLine(string.Equals(Configuration["Language"], "spanish", StringComparison.OrdinalIgnoreCase) ? "Buenos Dias." : "Good morning");
-                sb.AppendLine();
-
-                sb.AppendLine("Press any key to exit...");
-
-                display = sb.ToString();
+// Check for App Configuration endpoint
+string? endpoint = initialConfig["AppConfigEndpoint"];
+if (string.IsNullOrEmpty(endpoint))
+{
+    Console.WriteLine("App Configuration endpoint not found.");
+    Console.WriteLine("Please set the 'AppConfigEndpoint' environment variable to a valid Azure App Configuration endpoint URL and re-run this example.");
+    return 1;
+}
 
-                Console.Clear();
-                Console.Write(display);
-                
-                await Task.Delay(1000);                
-            } while (!token.IsCancellationRequested);
-        }
-    }
+// Connect to Azure App Configuration
+IConfigurationRefresher refresher = null!;
+builder.AddAzureAppConfiguration(options =>
+{
+    // Use DefaultAzureCredential for Microsoft Entra ID authentication
+    options.Connect(new Uri(endpoint), new DefaultAzureCredential())
+           // Load all keys that start with "Settings:" and have no label.
+           .Select("Settings:*")
+           .TrimKeyPrefix("Settings:")
+           .ConfigureRefresh(refreshOptions =>
+           {
+               // Reload configuration if any selected key-values have changed.
+               refreshOptions.RegisterAll()
+                             .SetRefreshInterval(TimeSpan.FromSeconds(10));
```

Comment:
Since we recently went with the default refresh rate, wondering if we should do it here.

Reply:
Makes sense. Updated.